tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +51,8 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	terminate := flag.Bool("terminate", true, "'true' to terminate an etcd cluster after testing.")
```

Comment:
I don't think this is a good way to do it because avoiding the etcd-tester orchestration code means reimplementing a lot of stuff (e.g., separate checker process, separate code to manage cluster, separate code to start/stop fuzzing, etc) and it won't preserve the same cluster across rounds masking long-run bugs. Can namazu be controlled with a `failure` wrapper in `etcd-tester` that calls out to a script on `Inject` and `Recover` to start/stop the fuzzing?